src/components/Upload/Cropper/EasyCrop.tsx

```diff
@@ -165,14 +174,28 @@ const EasyCrop = forwardRef<EasyCropHandle, EasyCropProps>((props, ref) => {
               iconProps={{
                 path: IconName.mdiPlus,
               }}
-              onClick={() => setZoomVal(zoomVal + ZOOM_STEP)}
+              onClick={() => {
+                setButtonType('in');
+                setZoomVal(zoomVal + ZOOM_STEP);
+              }}
               shape={ButtonShape.Round}
               size={ButtonSize.Small}
               theme={mergedTheme}
               themeContainerId={themeContainerId}
               variant={ButtonVariant.SystemUI}
             />
           </Stack>
+          <div
+            className={styles.visuallyhidden}
+            aria-live="assertive"
+            role="status"
+          >
+            {buttonType === 'in'
+              ? `${zoomInButtonAriaLabelText} ${parseFloat(zoomVal.toFixed(2))}`
+              : `${zoomOutButtonAriaLabelText} ${parseFloat(
+                  zoomVal.toFixed(2)
+                )}`}
```

Comment:
may be a good idea to separate the button type into two so we aren't updating this string when left/right is interacted with
alternatively, could make this empty when left or right